services/gateway/apis/swagger/version2/apiSchema.json

```diff
@@ -0,0 +1,2222 @@
+			"minLength": 1
+		},
+		"sender": {
+			"name": "sender",
+			"in": "query",
+			"description": "Lisk address, public key or account name (senderId)",
+			"type": "string",
+			"minLength": 1
+		},
+		"transactionData": {
+			"name": "data",
+			"in": "query",
+			"description": "Search in transaction data field. Exact match, but can use % as wildcard e.g. data=test%",
+			"type": "string",
+			"minLength": 1
+		},
+		"moduleAssetId": {
+			"name": "moduleAssetId",
+			"in": "query",
+			"description": "Combination of module:asset Id",
+			"type": "string",
+			"minLength": 1
+		},
+		"moduleAssetName": {
+			"name": "moduleAssetName",
+			"in": "query",
+			"description": "Combination of module:asset Name",
+			"type": "string",
+			"minLength": 1
+		},
+		"amount": {
+			"name": "amount",
+			"in": "query",
+			"description": "Amount range of from:to amount",
+			"type": "integer",
+			"minimum": 0
+		},
+		"timestamp": {
+			"name": "timestamp",
+			"in": "query",
+			"description": "Range fromTimestamp:toTimestamp",
+			"type": "string",
+			"minimum": 0
+		},
+		"storeName": {
+			"name": "storeName",
+			"in": "path",
+			"description": "Name of the store container",
+			"required": true,
+			"type": "string"
+		},
+		"storeId": {
+			"name": "storeId",
+			"in": "path",
+			"description": "Id of the object in the container",
+			"required": true,
+			"type": "string"
+		},
+		"marketDataProvider": {
+			"name": "provider",
+			"in": "path",
+			"description": "Source of the news among different data providers (Poloniex, Bittrex, etc.)",
+			"required": true,
+			"type": "string",
+			"enum": [
+				"poloniex",
+				"bittrex"
+			]
+		},
+		"status": {
+			"name": "status",
+			"in": "query",
+			"description": "Delegate account status",
+			"type": "string",
+			"minLength": 1
+		},
+		"isDelegate": {
+			"name": "isDelegate",
+			"in": "query",
+			"description": "Filter delegate accounts",
+			"type": "string",
+			"minLength": 1
+		},
+		"transaction": {
+			"name": "transaction",
+			"in": "query",
+			"description": "Binary payload",
+			"type": "string",
+			"minLength": 1
+		},
+		"data": {
+			"name": "data",
+			"in": "query",
+			"description": "Query by asset data",
+			"type": "string",
+			"minLength": 1
+		},
+		"includePending": {
+			"name": "includePending",
+			"in": "query",
+			"description": "Retrieve pending transactions",
+			"type": "boolean",
+			"minLength": 1,
+			"default": false
+		}
+	},
+	"definitions": {
+		"Account": {
+			"type": "object",
+			"required": [
+				"address"
+			],
+			"properties": {
+				"summary": {
+					"type": "object",
+					"properties": {
+						"address": {
+							"type": "string",
+							"format": "address",
+							"example": "lskdwsyfmcko6mcd357446yatromr9vzgu7eb8y99",
+							"description": "The Lisk Address is the human readable representation of the accounts owners' public key.\nIt consists of multiple numbers followed by a capital 'L' at the end.\n"
```

Comment:
The description requires to be updated. It currently suits the older address format.